README.md

```diff
@@ -1 +1,57 @@
-# terra-enzyme-intl
\ No newline at end of file
+<!-- Logo -->
+<p align="center">
+  <img height="128" width="128" src="https://github.com/cerner/terra-enzyme-intl/raw/master/terra.png">
+</p>
+
+<!-- Name -->
+<h1 align="center">
+  Terra Enzyme Intl
+</h1>
+
+[![NPM version](https://badgen.net/npm/v/terra-enzyme-intl)](https://www.npmjs.org/package/terra-enzyme-intl)
+[![License](https://badgen.net/github/license/cerner/terra-enzyme-intl)](https://github.com/cerner/terra-enzyme-intl/blob/master/LICENSE)
+[![Build Status](https://badgen.net/travis/cerner/terra-enzyme-intl)](https://travis-ci.org/cerner/terra-enzyme-intl)
+[![Dependencies status](https://badgen.net/david/dep/cerner/terra-enzyme-intl)](https://david-dm.org/cerner/terra-enzyme-intl)
+[![devDependencies status](https://badgen.net/david/dev/cerner/terra-enzyme-intl)](https://david-dm.org/cerner/terra-enzyme-intl?type=dev)
+
+Enzyme helpers for fortifying tests that depend on react-intl by decoupling the need for actual translations. See: [react-intl documentation](https://github.com/yahoo/react-intl/wiki/Testing-with-React-Intl#helper-function-1)
+
+## Installation
+
+npm:
+
+```sh
+npm install --save-dev terra-enzyme-intl
+```
+
+or
+
+Yarn:
+
+```sh
+yarn add --dev terra-enzyme-intl
+```
+
+## Usage
+
+```jsx
+import { shallowWithIntl, mountWithIntl } from 'terra-enzyme-intl';
+
+const shallowWrapper = shallowWithIntl(<CustomComponent />);
+const mountWrapper = mountWithIntl(<CustomComponent />);
```

Comment:
We need documentation on usage for each exported option. It looks like method accepts and element, context, ect. We should also document the mockIntl we are exporting

Reply:
For sure! I'll add em